Projects/UOContent/Mobiles/Monsters/AOS/Revenant.cs

```diff
@@ -175,19 +173,9 @@ public override bool OnBeforeDeath()
             return false;
         }
 
-        public override void Serialize(IGenericWriter writer)
+        [AfterDeserialization]
```

Comment:
Change this to have the false argument.

Reply:
`cs
        [AfterDeserialization(false)]
        private void AfterDeserialization()
        {
            Delete();
        }
`
like that?

Reply:
Yes!